distribution/docs/src/main/jdocs/content/_plugins/metacard-groomer-contents.adoc

```diff
@@ -0,0 +1,31 @@
+:type: plugin
+:status: published
+:title: Metacard Groomer
+:link: _metacard_groomer
+:plugintypes: preingest
+:summary: Modifies metacards when created or updated.
+
+The Metacard Groomer Pre-Ingest plugin makes modifications to `CreateRequest` and `UpdateRequest` metacards.
+
+Use this pre-ingest plugin as a convenience to apply basic rules for your metacards. 
```

Comment:
Wonder if we should make a note for this plugin that issues could arise if this plugin is turned off because parts of the system are counting on it running.